examples/react-map-gl-draw/app.js

```diff
@@ -77,7 +77,9 @@ export default class App extends Component {
         <Editor
           ref={_ => (this._editorRef = _)}
           clickRadius={12}
-          onSelect={selected => this.setState({ selectedFeatureIndex: selected ? 0 : null })}
+          onSelect={selected => {
+            this.setState({ selectedFeatureIndex: selected && selected.selectedFeatureIndex });
```

Comment:
- When is `selected` ever null?
- Use `selectedFeature` instead?

Reply:
- `selected` will be null when user click empty space while a feature is selected, or a polygon is finished. 
- why should use `selectedFeature`?

Reply:
If the callback argument also contains `mapCoords` and `screenCoords`, then it should always be a valid object even if nothing is selected.

Reply:
I am not sure what you mean, but right now, `onSelect` will be triggered in the following cases
- a polygon is finished
- a feature is selected
- click empty space while a feature is selected

Reply:
According to your documentation, the callback parameter also provides information about the click itself, which should still be accessible even if no feature is selected.